src/theme/components/label.ts

```diff
@@ -11,6 +11,9 @@ const Label = defineStyleConfig({
     ...labelLegendText,
     flex: props.isInlined ? "1" : null,
     whiteSpace: props.isInlined ? "nowrap" : null,
+    span: {
+      fontWeight: "light",
```

Comment:
The PR description says "medium" but this is "light". Is this correct?

Reply:
Same here.